src/hooks.ts

```diff
@@ -87,6 +87,10 @@ interface UseDecision {
   ];
 }
 
+interface UseTrackEvents {
```

Comment:
NIT: singular naming maybe?

```suggestion
interface UseTrackEvent {
```